pkgs/desktops/gnome/extensions/gsconnect/default.nix

```diff
@@ -22,15 +22,15 @@
 
 stdenv.mkDerivation rec {
   pname = "gnome-shell-extension-gsconnect";
-  version = "58";
+  version = "4889447";
 
   outputs = [ "out" "installedTests" ];
 
   src = fetchFromGitHub {
     owner = "GSConnect";
     repo = "gnome-shell-extension-gsconnect";
     rev = "v${version}";
```

Comment:
Please use full commit id `48894474e9af5375731c2e63c64c4c464890b30a` as the rev. Otherwise, there might be a collision in the future.

Reply:
Or probably will want https://github.com/GSConnect/gnome-shell-extension-gsconnect/commit/7c387edc71cf7461641d228c455a0bff9ec3202e, since that includes extra validation.